### PR TITLE
Making sentence_order_label optional

### PR DESCRIPTION
**Pull request by Koziev**
1) Making sentence_order_label optional in LeanAlbertForPreTraining.forward to allow calculation of loss without sentence_order_label, e.g. for quasiperplexity.

2) Adding dtype=int when creating attention_mask to prevent an assertion error.

### CODE REVIEW

**Review by mryab — APPROVED** [review]
Thanks for your PR! Technically, the assertion should not run if we're creating the attention mask inside `forward`; I'll fix that in a follow-up commit
